### PR TITLE
Added current committee members to About Us page

### PR DESCRIPTION
**Pull request by clyne**

### CODE REVIEW

File: about.md [hunk 1]
```diff
@@ -10,6 +10,36 @@ to serve on the ESDS Leadership Committee and/or Working Groups as
...
+### Community Working Group
+* Julia Kent, CISL
+* Katelyn FitzGerald, CISL
+* Elena Romashkova, CGD
+
+### Events Working Group
+* Daniel Adriaansen, RAL
+* John Clyne, CISL
+* Orhan Eroglu, CISL
+* Katelyn Fitzgerald, CISL
```
Comment (kafitzgerald) [inline about.md:36]:
```suggestion
* Katelyn FitzGerald, CISL
```

**Review by kafitzgerald — APPROVED** [review]

**Review by TeaganKing — APPROVED** [review]
Very minor comment, but maybe it would be nice to alphabetize these unless there's a reason for the current ordering? Otherwise looks good to me.

File: about.md [hunk 2]
```diff
@@ -10,6 +10,36 @@ to serve on the ESDS Leadership Committee and/or Working Groups as
...
+* Dan Adriaansen, RAL
+* Drew Camron, Unidata
+* John Clyne, CISL
+* Katie Dagon, CGD (stepping down, ~Feb 2024)
+* Orhan Eroglu, CISL
+* Teagan King, CGD
+* Falko Judt, MMM
+* Ryan May, Unidata
+* Mike Levy, CGD
+* Ulrike Romatschk, EOL
```
Comment (erogluorhan) [inline about.md:25]: Maybe a bit nit-picky, but the first half of it feels like alphabetically ordered (per last names). If that's the case, the order of the last names in the second half needs to be Judt, King, Levy, May, Romatschk instead.

File: about.md [hunk 3]
```diff
@@ -10,6 +10,36 @@ to serve on the ESDS Leadership Committee and/or Working Groups as
...
+
+### Community Working Group
+* Julia Kent, CISL
+* Katelyn FitzGerald, CISL
```
Comment (erogluorhan) [inline about.md:29]: Similar alphabetical nit-picking between Kent and FitzGerald :)

**Review by erogluorhan — APPROVED** [review]
Thanks for this! Approved, but two little comments though.

File: about.md [hunk 4]
```diff
@@ -10,6 +10,36 @@ to serve on the ESDS Leadership Committee and/or Working Groups as
 described in our [governance document](https://docs.google.com/document/d/1xjNzJqZjeRzUg5BMipQVUsy2xA1SRlEiyJLpPmUAs9E/edit?usp=sharing).
 
 
+## ESDS Committees and Working Groups
+
+### Leadership Committee
+* Dan Adriaansen, RAL
```
Comment (DanielAdriaansen) [inline about.md:16]:
```suggestion
* Daniel Adriaansen, RAL
```

Just to be consistent with below only. Dan is fine otherwise!

**Review by DanielAdriaansen — CHANGES_REQUESTED** [review]
I agree with any other alphabetizing recommendations, and request one change.

File: about.md (hunk 2 above)
Comment (rmshkv) [inline about.md:25]:
```suggestion
* Ulrike Romatschke, EOL
```
Fixing the spelling of Ulrike's name

Comment (clyne) [reply about.md]: Whoops. That's what happens when I multitask. Good catch. Thanks.

File: about.md (hunk 3 above)
Comment (clyne) [reply about.md:28]: Fixed. Thanks

**Comment by clyne** [comment]
> I agree with any other alphabetizing recommendations, and request one change.

Done!

**Review by katiedagon — COMMENTED** [review]
This is great, thanks for adding this @clyne. Do we want to add previous leadership members here? It could be a nice way to acknowledge the efforts of Deepak, Heather, Matt, Max, Kevin and others...

**Comment by clyne** [comment]
> This is great, thanks for adding this @clyne. Do we want to add previous leadership members here? It could be a nice way to acknowledge the efforts of Deepak, Heather, Matt, Max, Kevin and others...

Great idea! Can you think of who the "others" are?

**Comment by katiedagon** [comment]
> > This is great, thanks for adding this @clyne. Do we want to add previous leadership members here? It could be a nice way to acknowledge the efforts of Deepak, Heather, Matt, Max, Kevin and others...
> 
> Great idea! Can you think of who the "others" are?

Here is everyone I can think of who is not already listed under the current committee membership (others feel free to chime in if I missed anyone):
Anderson Banihirwe (CISL), Deepak Cherian (CGD), Heather Craker (CISL), Max Grover (CGD), Thomas Hauser (CISL), Matt Long (CGD), Kevin Paul (CISL)

**Comment by clyne** [comment]
> > > This is great, thanks for adding this @clyne. Do we want to add previous leadership members here? It could be a nice way to acknowledge the efforts of Deepak, Heather, Matt, Max, Kevin and others...
> > 
> > 
> > Great idea! Can you think of who the "others" are?
> 
> Here is everyone I can think of who is not already listed under the current committee membership (others feel free to chime in if I missed anyone): Anderson Banihirwe (CISL), Deepak Cherian (CGD), Heather Craker (CISL), Max Grover (CGD), Thomas Hauser (CISL), Matt Long (CGD), Kevin Paul (CISL)

I've added the above members. Feel free to add others.

**Review by katiedagon — APPROVED** [review]

**Review by DanielAdriaansen — APPROVED** [review]

**Pull request merged**